src/main/java/apoc/load/Jdbc.java

```diff
@@ -221,11 +225,22 @@ public boolean hasNext() {
         }
 
         private Object convert(Object value) {
+
             if (value instanceof UUID || value instanceof BigInteger || value instanceof BigDecimal) {
                 return value.toString();
             }
             if (value instanceof java.util.Date) {
-                return ((java.util.Date) value).getTime();
+                if(!config.isEmpty()){
+                    Instant instant = Instant.ofEpochMilli(((Date) value).getTime());
+                    ZoneId timezone = config.containsKey("timezone") ? ZoneId.of("timezone") : ZoneId.of("UTC");
+                    if(config.containsKey("localDateTime") && (Boolean)config.get("localDateTime")) return instant.atZone(timezone).toLocalDateTime();
```

Comment:
Trasformiamo la mappa in una classe, dove ogni campo ha i valori di defaul a false come fatto in [ExportConfig](https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/0839f02317357b6c04f40f39f5c8ebfe0c924b53/src/main/java/apoc/export/util/ExportConfig.java)